assets/app/scripts/directives/nav.js

```diff
@@ -24,7 +24,7 @@ angular.module('openshiftConsole')
     return {
       restrict: 'EA',
       templateUrl: 'views/_project-nav.html',
-      link: function($scope, $elem) {
+        link: function($scope, $elem) {
```

Comment:
There is still whitespace getting added here, link should be at the same indent as templateUrl